Remoção do campo status e adição de migration com dropColumn

isabelakashima opened the pull request:
Foram removidas todas referências ao campo status, utilizado no modelo user.
Migration com dropColumn 'status' da tabela Users foi criada.

Fix #38

isabelakashima committed:
Remoção do campo status e adição de migration com dropColumn

Foram removidas todas referências ao campo status, utilizado no modelo user.
Migration com dropColumn 'status' da tabela Users foi criada.

Fix #38

isabelakashima committed:
Corrige erro ao editar registro no cadastro de um record

O botão editar estava dentro de um formulário, que causava o erro e foi eliminado.

Fix #40

ricardfo commented:
Foi adicionada a Trait da Senhaunicasocialite, mas não está funcionando, você migrou para versão 4? Quando eu tiro a trait eu consigo autenticar com a senha única.

isabelakashima committed:
Adiciona trait senha única USP

Foi configurada a atualização da biblioteca senhaunica-socialite para a versão 4

Fix #39
Fix #41

isabelakashima committed:
Adiciona arquivo composer.lock ao pull request

Fix #39
